### PR TITLE
Feature/facelift

### PR DESCRIPTION
**Pull request by LiorAgnin**
## Proposed changes

- Facelift

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: dapp/src/components/home/components/FeaturedCommunities/index.jsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  }
+}
+
+const mapStateToProps = state => ({
+  tokens: state.entities.tokens,
+  metadata: state.entities.metadata,
+  account: getAccountAddress(state),
+  foreignNetwork: getForeignNetwork(state),
+  networkType: state.network.networkType,
+  ...state.screens.oven
```
Comment (leonprou) [inline dapp/src/components/home/components/FeaturedCommunities/index.jsx]: should rename the `oven` to better name like `home`

File: dapp/src/reducers/accounts.js [hunk 2]
```diff
@@ -58,6 +58,9 @@ const handlers = {
   },
   [FETCH_TOKEN_LIST.SUCCESS]: (state, action) => {
     return { ...state, tokens: action.response.result }
+  },
+  [actions.FETCH_COMMUNITIES.SUCCESS]: (state, action) => {
```
Comment (leonprou) [inline dapp/src/reducers/accounts.js]: try to put just the community addresses here

File: dapp/src/sagas/accounts.js [hunk 3]
```diff
@@ -64,12 +65,22 @@ function * watchBalanceOfToken ({ response }) {
   yield put(actions.balanceOfToken(response.tokenAddress, accountAddress))
 }
 
+function * fetchCommunities ({ account }) {
```
Comment (leonprou) [inline dapp/src/sagas/accounts.js]: I think you can use `createEntitiesFetch` to create this action

**Review by leonprou — CHANGES_REQUESTED** [review]
Looks good. very clean.
Some small comments, recommendations mostly.

Check please that bug with communityEntity override to see its behaviour

File: dapp/src/sagas/accounts.js (hunk 3 above)
Comment (LiorAgnin) [reply dapp/src/sagas/accounts.js]: Done

File: dapp/src/reducers/accounts.js (hunk 2 above)
Comment (LiorAgnin) [reply dapp/src/reducers/accounts.js]: Done

File: dapp/src/components/dashboard/pages/Businesses/index.jsx [hunk 4]
```diff
@@ -0,0 +1,273 @@
...
+            showProfile={() => this.showProfile(communityAddress, entity.account)}
+          />
+        ))
+    } else {
+      return <p className='entities__items__empty'>There are no any entities</p>
+    }
+  }
+
+  renderItems = () => {
+    const { transactionStatus, merchants } = this.props
```
[the remark this answers is not in the document]
Comment (LiorAgnin) [reply dapp/src/components/dashboard/pages/Businesses/index.jsx]: Done

File: dapp/src/components/dashboard/containers/MainDashboard/index.jsx [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+        </div>
+      </div>
+    )
+  }
+}
+
+const mapStateToProps = (state, { match }) => ({
+  accountAddress: getAccountAddress(state),
+  networkType: state.network.networkType,
+  token: state.entities.communities &&
```
[the remark this answers is not in the document]
Comment (LiorAgnin) [reply dapp/src/components/dashboard/containers/MainDashboard/index.jsx]: Already exist

File: dapp/src/components/dashboard/components/Header/index.jsx [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+import { isMobileOnly } from 'react-device-detect'
+import CommunityLogo from 'components/common/CommunityLogo'
+import { formatAddress } from 'utils/format'
+
+export default ({ tokenAddress, isClosed, name, networkType, token, metadata }) => {
+  return (
+    <div className='community_header'>
+      <div className='community_header__image'>
+        <CommunityLogo
+          isDaiToken={token && token.symbol === 'DAI'}
```
[the remark this answers is not in the document]
Comment (LiorAgnin) [reply dapp/src/components/dashboard/components/Header/index.jsx]: Done

File: dapp/src/components/common/ProfileDropDown/index.jsx [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+          <span className='cell small-24 profile__account__address'>
+            {formatAddress(accountAddress)}
+            <CopyToClipboard text={accountAddress}>
+              <FontAwesome name='clone' />
+            </CopyToClipboard>
+          </span>
+        </div>
+        {
+          !isEmpty(communitiesIOwn) && (
+            <React.Fragment>
```
[the remark this answers is not in the document]
Comment (LiorAgnin) [reply dapp/src/components/common/ProfileDropDown/index.jsx]: Done

**Review by leonprou — APPROVED** [review]
👍

**Pull request merged**